content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -224,6 +222,9 @@ different Kubernetes components.
 | `EnableEquivalenceClassCache` | - | Deprecated | 1.15 | - |
 | `ExperimentalCriticalPodAnnotation` | `false` | Alpha | 1.5 | 1.12 |
 | `ExperimentalCriticalPodAnnotation` | `false` | Deprecated | 1.13 | - |
+| `EvenPodsSpread` | `false` | Alpha | 1.16 | 1.17 |
+| `EvenPodsSpread` | `true` | Beta | 1.18 | 1.18 |
+| `EvenPodsSpread` | `true` | GA | 1.19 | - |
```

Comment:
Isn't the gate removed in 1.19?

Reply:
I think it serves as listing historical changes.

Reply:
@sftim for authoritative answer, but the historical purpose should be satisfied with the Alpha and Beta rows.

Reply:
This looks correct. Once a feature reaches GA, move the alpha+beta items to `stable/deprecated` table and add `GA or deprecated`.